src/styles/_product-list.scss

```diff
@@ -1,10 +1,14 @@
 .product-list {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
-  max-width: 950px;
   padding-bottom: 73px;
-  margin: 0 auto;
+
+  &__wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    
+    max-width: 950px;
+    margin: 0 auto;
```

Comment:
justify-content: center align items in the center of block, you don't need margin: 0 auto;

Reply:
+1

Reply:
Hi, @shyptia @ivan-kadykalo margin: 0, auto need to align to the center for my wrapper and not for items inside https://tinyurl.com/2ye8x5cu